challenge_configuration.yaml

```diff
@@ -1,20 +1,33 @@
+# Challenge details
 challenge_long_name: EFI-USGS River Chlorophyll Forecasting Challenge
+challenge_url: https://projects.ecoforecast.org/usgsrc4cast-ci
+github_repo: eco4cast/usgsrc4cast-ci
 project_id: usgsrc4cast
+
+# Endpoints
 endpoint: sdsc.osn.xsede.org
 noaa_endpoint: s3.flare-forecast.org
 submissions_endpoint: submit.ecoforecast.org
-scores_bucket: bio230014-bucket01/challenges/scores
+
+# Buckets
+archive_bucket: bio230014-bucket01/challenges/archive
+driver_bucket: bio230014-bucket01/challenges/drivers
```

Comment:
@rqthomas @cboettig I made this new drivers path in the `bio230014-bucket01` bucket because all I could find for bucket + path for drivers was hard coded in scripts called `bio230014-bucket01/neon4cast-drivers/noaa/gefs-v12` which wasn't consistent with the naming scheme elsewhere in the challenge_configuration.yaml

Reply:
Lets change it to `bio230014-bucket01/challenges/drivers/usgsrc4cast/noaa`